src/conduits/java/com/enderio/conduits/common/items/ConduitBlockItem.java

```diff
@@ -40,6 +40,13 @@ public InteractionResult place(BlockPlaceContext context) {
                 return InteractionResult.SUCCESS;
             }
         }
+        if (level.getBlockEntity(context.getHitResult().getBlockPos().relative(context.getHitResult().getDirection().getOpposite())) instanceof ConduitBlockEntity conduit) {
```

Comment:
This seemed to not work. The goal was to have it so you don't need to hit the hitbox, but can also hit the block underneath it, since the hitbox is small.